README.md

```diff
@@ -1 +1,170 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <br/>
+
+  <h3><b>LeaderBoard</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[LeaderBoard\] ](#-leaderboard-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [LeaderBoard] <a name="LeaderBoard: Project setup with WebPack"></a>
+
+**[LeaderBoard]**  website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+## 🛠 Built With <a name="Visual Studio Code"></a>
+
+### Tech Stack <a name="JavaScript HTML & CSS"></a>
+
+- HTML
+- CSS
+- Javascript ES6
+- Webpack
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Display scores**
+- **Add your Scores**
+- **API Services**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Coming Soon]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+If you are a new developer then you can learn Use callbacks and promises feature from this project.And also API Services
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Browser setup in your computer
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  https://github.com/ngeelani48/Leaderboard.git
+```
+
+### Install
+
+Install this project with:
+
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
+### Usage
+
+To run the project, execute the following command:
+- After Cloning this repo to your local machine
+- Open the `index.html` from dist folder in your browser
+- Open with Live Server
+### Run tests
+
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+npm install
+```
+Track HTML linter errors run:
+```
+npx hint .
```

Comment:
- [ ] [Optional] Kindly add these two steps here in getting started section to make it more professional:
  - Run the build using `npm run build` command.
  - Run the dev server using `npm run start` command.